libminifi/include/utils/file/FileUtils.h

```diff
@@ -541,6 +541,15 @@ class FileUtils {
     return child_path;
   }
 
+  static bool is_hidden(const std::string& path){
+#ifdef WIN32
+    DWORD attributes = GetFileAttributes(path.c_str());
```

Comment:
Please use `GetFileAttributesA` here. If you do not specify the A or the W version it depends on configuration (defines) which will be the default which can be accidentally set to W somewhere and we have a segfault on our hands.

Reply:
Good spot, thanks!